ocamltest/.depend

```diff
@@ -1,25 +1,27 @@
 run_unix.$(O): run_unix.c run.h ../runtime/caml/misc.h \
-  ../runtime/caml/config.h ../runtime/caml/m.h ../runtime/caml/s.h \
-  ../runtime/caml/camlatomic.h run_common.h
+ ../runtime/caml/config.h ../runtime/caml/m.h ../runtime/caml/s.h \
+ ../runtime/caml/camlatomic.h ../runtime/caml/misc.h run_common.h
 run_stubs.$(O): run_stubs.c run.h ../runtime/caml/misc.h \
-  ../runtime/caml/config.h ../runtime/caml/m.h ../runtime/caml/s.h \
-  ../runtime/caml/camlatomic.h ../runtime/caml/mlvalues.h \
-  ../runtime/caml/domain_state.h ../runtime/caml/domain_state.tbl \
-  ../runtime/caml/byte_domain_state.tbl ../runtime/caml/memory.h \
-  ../runtime/caml/gc.h ../runtime/caml/major_gc.h \
-  ../runtime/caml/minor_gc.h ../runtime/caml/addrmap.h \
-  ../runtime/caml/domain.h ../runtime/caml/alloc.h ../runtime/caml/io.h \
-  ../runtime/caml/platform.h ../runtime/caml/osdeps.h
+ ../runtime/caml/config.h ../runtime/caml/m.h ../runtime/caml/s.h \
+ ../runtime/caml/camlatomic.h ../runtime/caml/misc.h \
+ ../runtime/caml/mlvalues.h ../runtime/caml/domain_state.h \
+ ../runtime/caml/domain_state.tbl ../runtime/caml/byte_domain_state.tbl \
+ ../runtime/caml/memory.h ../runtime/caml/gc.h ../runtime/caml/mlvalues.h \
+ ../runtime/caml/major_gc.h ../runtime/caml/minor_gc.h \
+ ../runtime/caml/addrmap.h ../runtime/caml/domain.h \
+ ../runtime/caml/memory.h ../runtime/caml/alloc.h ../runtime/caml/io.h \
+ ../runtime/caml/platform.h ../runtime/caml/osdeps.h
 ocamltest_stdlib_stubs.$(O): ocamltest_stdlib_stubs.c \
-  ../runtime/caml/config.h ../runtime/caml/m.h ../runtime/caml/s.h \
-  ../runtime/caml/mlvalues.h ../runtime/caml/misc.h \
-  ../runtime/caml/camlatomic.h ../runtime/caml/domain_state.h \
-  ../runtime/caml/domain_state.tbl ../runtime/caml/byte_domain_state.tbl \
-  ../runtime/caml/memory.h ../runtime/caml/gc.h \
-  ../runtime/caml/major_gc.h ../runtime/caml/minor_gc.h \
-  ../runtime/caml/addrmap.h ../runtime/caml/domain.h \
-  ../runtime/caml/alloc.h ../runtime/caml/signals.h \
-  ../runtime/caml/osdeps.h
+ ../runtime/caml/config.h ../runtime/caml/m.h ../runtime/caml/s.h \
+ ../runtime/caml/mlvalues.h ../runtime/caml/config.h \
+ ../runtime/caml/misc.h ../runtime/caml/camlatomic.h \
+ ../runtime/caml/domain_state.h ../runtime/caml/domain_state.tbl \
+ ../runtime/caml/byte_domain_state.tbl ../runtime/caml/memory.h \
+ ../runtime/caml/gc.h ../runtime/caml/mlvalues.h \
+ ../runtime/caml/major_gc.h ../runtime/caml/minor_gc.h \
+ ../runtime/caml/addrmap.h ../runtime/caml/domain.h \
+ ../runtime/caml/memory.h ../runtime/caml/alloc.h ../runtime/caml/alloc.h \
```

Comment:
there is something odd going on here, some of these dependencies are duplicated, e.g. `../runtime/caml/alloc.h` and `../runtime/caml/mlvalues.h`

Reply:
Ok. This is beyond my knowledge. I propose to remove this commit so that you can look at it separately.

Reply:
Odd to see duplications, but are sure we didn't have duplicates before?